Documentation/ApiOverview/PasswordHashing/Index.rst

```diff
@@ -96,11 +96,12 @@ is pretty straight forward and helps you to compare passwords with their stored
 extensions.
 
 One last point on this basic hash knowledge section: Password hashes are always only as secure as
-the user submitted password: If a user has a trivial password like "foo", an attacker who got hold
+the chosen password: If a user has a trivial password like "foo", an attacker who has gotten hold
 of the salted password hash will always be successful to crack the hash with a common password hash
 crack tool, no matter how expensive the calculation is. Good password hashing does **not** rescue
-users from short passwords or simple passwords that can be found in a dictionary. It is usually a
-good idea to force users to register with a password that for instance at least has some minimum length.
+users from short or simple passwords that can be found in a dictionary. It is usually a
```

Comment:
I would rather leave it as is, as the simple passwords is bound to the dictionary, the short password not. Adding a comma makes it clearer:

```suggestion
users from short passwords, or simple passwords that can be found in a dictionary. It is usually a
```